QR-code-Genrator/main.py

```diff
@@ -0,0 +1,89 @@
+        lbl_emp_dept = Label(emp_Frame,text="  Department", font=("time new roman",15), bg="white").place(x=20,y=140)
+        lbl_emp_designation = Label(emp_Frame,text="  Designation", font=("time new roman",15), bg="white").place(x=20,y=180)
+
+
+        text_emp_code = Entry(emp_Frame, font=("time new roman",15), textvariable=self.var_emp_code, bg="lightyellow").place(x=200,y=60)
+        text_emp_name = Entry(emp_Frame, font=("time new roman",15), textvariable=self.var_name, bg="lightyellow").place(x=200,y=100)
+        text_emp_dept = Entry(emp_Frame, font=("time new roman",15), textvariable=self.var_department, bg="lightyellow").place(x=200,y=140)
+        text_emp_designation = Entry(emp_Frame, font=("time new roman",15), textvariable=self.var_designation, bg="lightyellow").place(x=200,y=180)
+
+        btn_genrator = Button(emp_Frame, text="QR Genrator", command=self.genrate, font=("time new roman", 15, "bold"), bg="#2196f3", fg="white").place(x=90, y=250, width=180, height="30")
+        btn_clear = Button(emp_Frame, text="Clear", command=self.clear, font=("time new roman", 15, "bold"), bg="#2196f3", fg="white").place(x=290, y=250, width=120, height="30")
+
+        self.msg=""
+        self.lbl_msg = Label(emp_Frame, text=self.msg, font=("time new roman",15), bg="white", fg="green")
+        self.lbl_msg.place(x=0,y=320, relwidth=1)
+       
+       
+        # Qr Code window design
+        qr_Frame=Frame(self.root,bd=2, relief=RIDGE,bg="white")
+        qr_Frame.place(x=600, y=100, width=250, height=380)
+
+        emp_title = Label(qr_Frame,text="Employee QR code", font=("goudy old style",15), bg="#FB9316", fg="white").place(x=0,y=0,relwidth=1)
+
+        self.qr_code = Label(qr_Frame, text="No QR\n available", font=("time new roman",15), bg="#D76C02", fg="white", bd=1, relief=RIDGE)
+        self.qr_code.place(x=35, y=100, width=180, height=180)
+
+    def clear(self):
+        self.var_emp_code.set('')
+        self.var_name.set('')
+        self.var_department.set('')
+        self.var_designation.set('')
+        self.msg=""
+        self.lbl_msg.config(text=self.msg)
+        self.qr_code.config(image='')
+        
+
+    def genrate(self):
+            if self.var_emp_code.get() == '' or self.var_name.get() == '' or self.var_department.get()  == '' or  self.var_designation.get() == '':
+                self.msg="All filed required !!!"
+                self.lbl_msg.config(text=self.msg, fg="red")
+
+            else:
+                qr_data=(f"Employee Id:{self.var_emp_code.get()}\nEmployee Name:{self.var_name.get()}\nDepartment:{self.var_department.get()}\nDesignation:{self.var_designation.get()}")
+                qr_code=qrcode.make(qr_data)
+                # print(qr_code)
+                qr_code=resizeimage.resize_cover(qr_code,[180,180])
+                qr_code.save('employee_qr/emp_'+str(self.var_emp_code.get()+'.png'))
```

Comment:
Why haven't you updated the paths even after mentioning??
```suggestion
                qr_code.save('./QR-code-Genrator/employee_qr/emp_'+str(self.var_emp_code.get()+'.png'))
```